packages/composable-controller/CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.0.1]
+### Changed
+- Relax dependency on `@metamask/controller-utils` (use `^` instead of `~`) ([#998](https://github.com/MetaMask/controllers/pull/998))
+- Align with changes to `@metamask/network-controller`'s `NetworkState` type (`provider` -> `providerConfig`) ([#995](https://github.com/MetaMask/controllers/pull/995))
```

Comment:
Looks like this was only a change to the test module, so this can be omitted

```suggestion
```

Reply:
Removed in [a24e1cc](https://github.com/MetaMask/controllers/pull/999/commits/a24e1ccf93fe97dd79dcba59200b3a43bdd4bf95).